app/presenters/publishing_api_presenters/edition.rb

```diff
@@ -73,5 +81,20 @@ def default_update_type
     def specialist_sectors
       [edition.primary_specialist_sector_tag].compact + edition.secondary_specialist_sector_tags
     end
+
+    def access_limited
+      {
+        users: users.map(&:id)
+      }
+    end
+
+    def users
+      return @users if @users
+
+      org_users = edition.organisations.map(&:users).flatten
+      all_users = (org_users + edition.authors)
```

Comment:
I was basing this on the test here: https://github.com/alphagov/whitehall/blob/re-enable-access-limiting-on-case-studies/test/unit/edition/limited_access_test.rb#L62

Do you think that might be specific to world locations?